tests/sqllogictests/suites/query/union.test

```diff
@@ -177,10 +177,3 @@ drop table t1
 
 statement error 1065
 select [1,2,3] union all select 2
-
-# type coercion
```

Comment:
Why delete the test?

Reply:
Oh, I overlooked the result of the git revert.